packages/server/src/fhir/operations/execute.ts

```diff
@@ -172,6 +172,25 @@ async function runInLambda(request: BotExecutionRequest): Promise<BotExecutionRe
   }
 }
 
+/**
+ * Returns the AWS Lambda function name for the given bot.
+ * By default, the function name is based on the bot ID.
+ * If the bot has a custom function, and the server allows it, then that is used instead.
+ * @param bot The Bot resource.
+ * @returns The AWS Lambda function name.
+ */
+export function getLambdaFunctionName(bot: Bot): string {
+  if (getConfig().botCustomFunctionsEnabled) {
+    const customFunction = bot.identifier?.find((id) => id.system === 'https://medplum.com/custom-function');
```

Comment:
I think you can just use our `getIdentifier` helper here: 
https://www.medplum.com/docs/sdk/modules#getidentifier